spec/system/claims/sign_in_as_a_claims_user_spec.rb

```diff
@@ -142,4 +242,30 @@ def i_do_not_have_access_to_support_page
   def then_i_am_redirect_to_internal_server_error
     expect(page).to have_content("Sorry, there’s a problem with the service")
   end
+
+  def then_i_dont_get_redirected_to_support_organisations
+    expect(page).to have_no_current_path claims_support_claims_path, ignore_query: true
+  end
+
+  def then_i_see_a_list_of_organisations
+    expect(page).to have_current_path claims_support_schools_path, ignore_query: true
+    expect(page).to have_content("Claims School")
+  end
+
+  def and_i_see_an_empty_schools_page
+    expect(page).to have_current_path(claims_schools_path)
+    expect(page).to have_content("Organisations")
+  end
```

Comment:
This is an awkward test because there is no "empty state" for this page in the Claims service that we can positively assert on 😕

How about something like this, to test the page **does not have links**?

```suggestion
  def and_i_see_an_empty_schools_page
    expect(page).to have_current_path(claims_schools_path)
    expect(page).to have_content("Organisations")
    expect(find("#main-content")).not_to have_link
  end
```